controller.go

```diff
@@ -115,6 +115,7 @@ func (c *Controller) Init(ctx *context.Context, controllerName, actionName strin
 	c.EnableXSRF = true
 	c.Data = ctx.Input.Data()
 	c.methodMapping = make(map[string]func())
+	c.CruSession = ctx.Input.CruSession
```

Comment:
Outside package can  directly acces  `CruSession‘   ,so set value in init function. and  ctx.Input.CruSession is `nil` if SessionOn = false .

I argree "no need assign again in Controller.StartSession() " , but we need keep this function for old system.